src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,40 @@
+/* eslint-disable prettier/prettier */
+/* eslint-disable no-unused-expressions */
+/* eslint-disable no-sequences */
+/* eslint-disable prettier/prettier */
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+
+  constructor(name, health = 100) {
+    (this.name = name), (this.health = health), Animal.alive.push(this);
+  }
+
+  checkHealth() {
+    if (this.health <= 0) {
+      Animal.alive = Animal.alive.filter((animal) => animal !== this);
+    }
+  }
```

Comment:
The `checkHealth` method is not necessary. Instead, you can directly filter the `Animal.alive` array inside the `bite` method after decreasing the health of the herbivore.